components/sync15/src/request.rs

```diff
@@ -69,7 +70,7 @@ impl CollectionRequest {
     where
         V: Into<Vec<String>>,
```

Comment:
Changing this to V: Into<Vec<Guid>> is probably better.

Reply:
Do you mean changing it to `V: into<Vec<Guid>>`? The compiler says 1 type argument expected on `Vec`.

Reply:
Whoops, github for some reason removed the inner brackets because it didn't have code quotes, sorry!

Reply:
Done!